registration/src/main/java/com/kyriba/registration/api/dto/AttendeeRegistrationRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.kyriba.registration.api.dto;
+
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@ApiModel(description = "Request model for attendee registration request")
+@Data
+@NoArgsConstructor
+public class AttendeeRegistrationRequest
+{
+  @ApiModelProperty(value = "Attendee First name")
+  private String firstName;
+
+  @ApiModelProperty(value = "Attendee Last name")
```

Comment:
Document only those fields that are not obvious and provide example to generate sample data for Swagger UI.

Reply:
ok